doc/radosgw/admin.rst

```diff
@@ -476,23 +476,40 @@ commands. ::
 Rate Limit Management
 =====================
 
-The Ceph Object Gateway enables you to set rate limits on users and buckets. 
-Rate limit includes the maximum number of read ops and write ops per minute
-and how many bytes per minute could be written or read per user or per bucket.
-Requests that are using GET or HEAD method in the REST request are considered as "read requests", otherwise they are considered as "write requests".
-Every Object Gateway tracks per user and bucket metrics separately, these metrics are not shared with other gateways.
-That means that the desired limits configured should be divide by the number of active Object Gateways.
-For example, if userA should be limited by 10 ops per minute and there are 2 Object Gateways in the cluster,
-the limit over userA should be 5 (10 ops per minute / 2 RGWs).
-If the requests are **not** balanced between RGWs, the rate limit may be underutilized.
-For example, if the ops limit is 5 and there are 2 RGWs, **but** the Load Balancer send load only to one of those RGWs,
-The effective limit would be 5 ops, because this limit is enforced per RGW.
-If there is a limit reached for bucket not for user or vice versa the request would be cancelled as well.
-The bandwidth counting happens after the request is being accepted, as a result, even if in the middle of the request the bucket/user has reached its bandwidth limit this request will proceed.
-The RGW will keep a "debt" of used bytes more than the configured value and will prevent this user/bucket from sending more requests until there "debt" is being paid.
-The "debt" maximum size is twice the max-read/write-bytes per minute.
-If userA has 1 byte read limit per minute and this user tries to GET 1 GB object, the user will be able to do it.
-After userA completes this 1GB operation, the RGW will block the user request for up to 2 minutes until userA will be able to send GET request again.
+The Ceph Object Gateway makes it possible to set rate limits on users and
+buckets.  "Rate limit" includes the maximum number of read operations (read
+ops) and write operations (write ops) per minute and the number of bytes per
+minute that can be written or read per user or per bucket.
+
+Requests that use the ``GET`` method or the ``HEAD`` method in their REST
+requests are "read requests". All other requests are "write requests".  
```

Comment:
Since we speak of "operations" above, should we s/requests/operations/?

Reply:
I think that's fair. I'll make the change.

Reply:
Actually... I don't know about this. The sentence "All other requests are 'write requests'" is not the same as the sentence "All other operations are 'write requests'." It seems to me that there is a mismatch of scope in this latter sentence.

Reply:
I've changed the first instance of "Requests" to "operations", but left the others. This should give the effect of transitioning from operations in general to specific read and write requests.

Reply:
I think that it's correct in general to use "request" when referring to a message cycle in the HTTP protocol, specifically.  Internally, the rgw server handles most of these via a lower-level, internal unit of work called an "op" (RGWOp).  Unless something specific in the RGW internal server workflow--below the HTTP layer--is being referred to, the term "operation" might be at least somewhat confusing.